src/Fixer/ControlStructure/TrailingCommaInMultilineFixer.php

```diff
@@ -98,17 +96,6 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['array'])
                 ->setAllowedValues([new AllowedValueSubset([self::ELEMENTS_ARRAYS, self::ELEMENTS_ARGUMENTS, self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS])])
                 ->setDefault([self::ELEMENTS_ARRAYS])
-                ->setNormalizer(static function (Options $options, array $value) {
-                    if (\PHP_VERSION_ID < 8_00_00) { // @TODO: drop condition when PHP 8.0+ is required
-                        foreach ([self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS] as $option) {
-                            if (\in_array($option, $value, true)) {
-                                throw new InvalidOptionsForEnvException(sprintf('"%s" option can only be enabled with PHP 8.0+.', $option));
-                            }
-                        }
-                    }
-
-                    return $value;
-                })
```

Comment:
please elaborate on reasoning.

did you had a chance to compare with other rules with similar concern?
I would love to have overview and align on consensus for all rules (perhaps out of scope of this PR)

Reply:
I think it might be better for end users to always allow the `parameters` config and simply ignore it on PHP 7.4. But yeah that will require to reconsider our approach globally. Let's keep current rule level behavior. The rule should be configured automatically at ruleset level.

Reply:
> reconsider our approach globally

i think it may be interesting part to go with. Would you mind to create follow-up ticket to have that investigation?

Reply:
Sure: #7919.